server/boards/app/files.go

```diff
@@ -90,26 +69,17 @@ func (a *App) GetFileInfo(filename string) (*mm_model.FileInfo, error) {
 }
 
 func (a *App) GetFile(teamID, rootID, fileName string) (*mm_model.FileInfo, filestore.ReadCloseSeeker, error) {
-	fileInfo, err := a.GetFileInfo(fileName)
-	if err != nil && !model.IsErrNotFound(err) {
-		a.logger.Error("111")
+	fileInfo, filePath, err := a.GetFilePath(teamID, rootID, fileName)
+	if err != nil {
+		a.logger.Error(fmt.Sprintf("GetFile: Failed to GetFilePath. Team: %s, board: %s, filename: %s, error: %e", teamID, rootID, fileName, err))
```

Comment:
Nit:  I will do a follow-up PR to log using the mlog fields instead of sprintf.

Reply:
Ha, just realized you are probably talking about product wide, not just this PR. I took care of this PR. :)